media-classification/src/main/resources/nl/vpro/domain/media/classification/ebu_ContentGenreCS.xml

```diff
@@ -209,19 +306,153 @@
         <Name xml:lang="nl">Muziek</Name>
         <Reference>urn:tva:metadata:cs:2004:05</Reference>
         <Reference>urn:mis:genre:MUSIC</Reference>
-        <Reference>urn:bindinc:genre:Muziek</Reference>
-        <Reference>urn:bindinc:genre:Musica</Reference>
-        <Reference>urn:bindinc:genre:Musique</Reference>
+        <Reference>urn:mis:bindinc:Muziek</Reference>
+        <Reference>urn:mis:bindinc:Musica</Reference>
         <ValidityFlag>1</ValidityFlag>
```

Comment:
Wat is dit? Waarom heb je aan de bindinc references gezeten?

Zo gaat de bindinc-import kapot.